src/main/kotlin/org/rust/lang/core/grammar/rust.bnf

```diff
@@ -758,7 +755,7 @@ private path_generic_args_without_colons_part_inner ::= ident_or_self_or_super [
 //    <T>::c<V,W>
 //    <T as Foo>::c(V) -> W
 //
-private meta qualified_path ::= '<' type [ AS trait_ref ] '>' '::' << param >>
+private meta qualified_path ::= '<' type [ AS trait_ref ] '>' '::' << fn_param >>
```

Comment:
Maybe we should rename `*param` to `*parameter`? I'd rather use long names for things which do not have corresponding Rust term.